config/develop.json

```diff
@@ -9,5 +9,17 @@
     "max": 20,
     "idleTimeoutMillis": 1000,
     "connectionTimeoutMillis": 5000
+  },
+
+  "cardanoExplorerDb": {
+    "user": "postgres",
+    "host": "localhost",
+    "database": "cexplorer",
+    "password": "mysecretpassword",
+    "port": 5432,
```

Comment:
Can we please have the cardano explorer variables configurable by environmental variables too?